config/software/ruby-windows.rb

```diff
@@ -25,29 +24,43 @@
     relative_path "ruby-#{version}-i386-mingw32"
     source url: "https://s3-us-west-2.amazonaws.com/yakyakyak/ruby-#{version}-i386-mingw32.7z"
 
+    default_version "2.0.0-p647"
     version("2.0.0-p647") { source md5: "0b1e8f16580f26fd0992fad3834cb83d" }
   end
 elsif windows_arch_i386?
+  default_version "2.0.0-p648"
```

Comment:
No 2.3.  2.3 has no ffi support on windows.  2.1.8 will build but there was one minor issue with spec/functional/resource/remote_file.rb that I think I've fixed - so you can attempt that version.